esp/esp/program/controllers/classchange.py

```diff
@@ -100,7 +100,7 @@ def get_unchanged_student_email_text(self, student_ind, for_real = False):
             text += "%s\n\n<br /><br />\n\n" % self.get_student_schedule(student_ind, for_real)
             text += "See you soon!<br /><br />"
         else:
-            text += "We're sorry that we couldn't accomodate your class preferences this time, and we hope to see you at a future ESP program.<br /<br />\n\n"
+            text += "We're sorry that we couldn't accommodate your class preferences this time, and we hope to see you at a future self.program.niceName() program.<br /<br />\n\n"
```

Comment:
Just putting the function call inside the string here doesn't work; you'll need to either do it with `+` like on the next line, or with `%` like on line 100.

Reply:
Thanks, will change.